### PR TITLE
vxWorks adding few errnoLib related constants.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by tgross35** [comment]
https://github.com/rust-lang/rust/pull/128751 mentions `VX_TASK_NAME_LEN`. Is that something defined in libc?

**Comment by devnexen** [comment]
I m unsure it s possible yet due to the uncertainty of its value unless a minimum vxworks release is supported.

**Comment by tgross35** [comment]
It may be worth specifying a minimum version, even if old, to make it obvious which APIs can be relied upon. If you are up for this, just make a PR to document some minimum version in https://github.com/rust-lang/rust/blob/60d146580c10036ce89e019422c6bc2fd9729b65/src/doc/rustc/src/platform-support/vxworks.md. Then we can ping a handful of people who have been involved in VxWorks to get their feedback.

**Review by biabbas — CHANGES_REQUESTED** [review]

File: src/vxworks/mod.rs
Comment (devnexen) [reply src/vxworks/mod.rs:728]: they do not exist, see they re not exposed it s only to server as base for constants such as : 

pub const S_taskLib_NAME_NOT_FOUND: ::c_int = `taskErrorBase + 0x0065;`

Comment (biabbas) [reply src/vxworks/mod.rs:728]: Ok. Can you change the comments. None constants are defined in errnoLib.h

Comment (devnexen) [reply src/vxworks/mod.rs:728]: sure

File: src/vxworks/mod.rs
Comment (biabbas) [inline src/vxworks/mod.rs:734]: These are defined in taskLibCommon.h

File: src/vxworks/mod.rs
Comment (biabbas) [inline src/vxworks/mod.rs:739]: These are defined in semLibCommon.h

File: src/vxworks/mod.rs
Comment (biabbas) [inline src/vxworks/mod.rs:745]: These in objLibCommon.h

**Review by biabbas — CHANGES_REQUESTED** [review]
Are these defined somewhere else in your VxWorks reference?

File: src/vxworks/mod.rs
Comment (devnexen) [reply src/vxworks/mod.rs:734]: oh I see I confused with vxWorks 6. fixing.

**Review by biabbas — COMMENTED** [review]
Looks good now. Not sure Why CI tests failed. Maybe reshuffling the constants so that their indentation increases progressively.

**Comment by tgross35** [comment]
CI has some known problems. We're working on getting those sorted out (will need to do that before merge)

**Review by biabbas — APPROVED** [review]

**Comment by tgross35** [comment]
> I m unsure it s possible yet due to the uncertainty of its value unless a minimum vxworks release is supported.

Does that work now with the minimum version of 7?

**Comment by devnexen** [comment]
it had been added in the last commit

**Comment by tgross35** [comment]
Not going to block on this since it is tier 3, but it may be worth adding a vxworks file in `libc-test/semver`.

**Pull request merged**
